### DISCUSSION TITLE
关于字段类型

### DISCUSSION THREAD

**Discussion by lengyuxuan** [apolloconfig/apollo#4716] [Ideas]
我惊喜的发现 docker hub 上的  2.1.0-SNAPSHOT 已经支持了字段类型：

![image](https://user-images.githubusercontent.com/23334732/215934968-d5632128-8d92-475c-a9ca-3533fa26e4bf.png)

但当我尝试更新 sdk 时却发现 api 尚未支持，得到的返回结果未区分类型：
![image](https://user-images.githubusercontent.com/23334732/215935055-372b7986-ac96-4360-8fd0-ff627c25a214.png)

我查看了提交记录 #4542 ，已经是几个月前实现的了，请问计划在哪个版本提供 api 的支持呢？

## ps
将来计划的获取接口得到的结果是这样解析好的：

```json
{
  "configurations": {
    "number": 15,
    "bool":  false,
    "json": {"a":123}
  }
}
```
还是：
```json
{
  "configurations": {
    "number": {
      "type": 1,
      "value": "15"
    },
    "bool": {
      "type": 2,
      "value": "boolean"
    },
    "json": {
      "type": 3,
      "value": "{\"a\":123}"
    }
  }
}
```

**Comment by nobodyiam** [comment]
Changing the API interface has a relatively large impact on the existing users, so we first support the type feature on the UI. 
How to make the client perceive the type needs to be discussed carefully by the community.
